frontend/src/locales/en.yml

```diff
@@ -1168,7 +1176,9 @@ label:
     description: >-
       A label is a key-value pair that helps you identify the tenant for a
       database. {link}.
-  db-name-template-tips: Allow you to group databases with the same {placeholder} from different tenants. {link}.
+  db-name-template-tips: >-
```

Comment:
Please exclude these formatting changes.

@LiuJi-Jim FYI

Reply:
@tianzhou I'm also annoyed and  working on it. T_T